src/NzbDrone.Core/Update/UpdateCheckService.cs

```diff
@@ -46,7 +47,7 @@ public UpdatePackage AvailableUpdate()
                 try
                 {
                     _logger.Info("Branch [{0}] is being redirected to [{1}]]", _configFileProvider.Branch, latestAvailable.Branch);
-                    var config = _configFileProvider.GetConfigDictionary();
+                    var config = new Dictionary<string, object>();
                     config["Branch"] = latestAvailable.Branch;
```

Comment:
we should write a unit-test for `ConfigFileProvider` to make sure it will do what this excepts it to do.